interface/patient_tracker/bootstrap_submenu.js

```diff
@@ -0,0 +1,43 @@
+/****  FUNCTIONS RELATED TO NAVIGATION *****/
```

Comment:
TODO (for me)
Add a license/author/copyright header

Reply:
Actually, I don't see this script being used anywhere.
@ophthal , can I remove this script (interface/patient_tracker/bootstrap_submenu.js); I can not find it being used anywhere?

Reply:
You are correct.  All these functions were merged into messages/js/reminder_appts.js.
This file can be deleted.

Reply:
awesome. I'll delete this then.